ogc/tiles/templates/tiles.go.html

```diff
@@ -51,9 +54,8 @@ <h1 class="title">{{ .Config.Title }} - {{ i18n "Tiles" .Language }}</h1>
           <td id="field-srs">{{ $defaultSrs.Srs }}</td>
         </tr>
         <tr>
-          <th>Metadata Tiles</th>
-          <td id="field-metadata"><a id="href-metadata"
-                                     href="tiles/{{ get $projections $defaultSrs.Srs }}">Link</a>
+          <th>Metadata</th>
+          <td id="field-metadata"><a id="href-metadata" href="tiles/{{ get $projections $defaultSrs.Srs }}">Bekijk {{ get $projections $defaultSrs.Srs }} tiles metadata</a>
```

Comment:
Muggenzifterij, maar misschien deze 'Bekijk' meteen nog even in de i18n gooien?

Reply:
doe ik, terecht